data/sql/unit_of_measure.sql

```diff
@@ -3,95 +3,95 @@
 INSERT INTO "unit_of_measure" VALUES('EPSG','1024','bin','scale',1.0,NULL,0);
 INSERT INTO "unit_of_measure" VALUES('EPSG','1025','millimetre','length',0.001,NULL,0);
 INSERT INTO "unit_of_measure" VALUES('EPSG','1026','metre per second','length',1.0,NULL,0);
-INSERT INTO "unit_of_measure" VALUES('EPSG','1027','millimetres per year','length',3.16887651727314875889e-11,NULL,0);
+INSERT INTO "unit_of_measure" VALUES('EPSG','1027','millimetres per year','length',3.16887651727314922922e-11,NULL,0);
```

Comment:
Looks like all floating point values with lots of decimals changed (not for 'US survey foot').
Just curiosity: did it actually change in the EPSG db, or is it due to any operation in our side?

Reply:
I did use Python3, not sure if that is a factor.